ClangSharpPInvokeGenerator/CXCursorExtensions.cs

```diff
@@ -1,329 +0,0 @@
-﻿using System.Diagnostics;
-using ClangSharp;
-
-namespace ClangSharpPInvokeGenerator
```

Comment:
These all got moved into `CursorWriter` so they could depend on the `ConfigurationOptions`